app/assets/stylesheets/components/styled-box.scss

```diff
@@ -13,7 +13,7 @@
     background-position: center center;
     cursor: pointer;
     overflow: hidden;
-
+ 
```

Comment:
I think your text editor added extra white space. Would you consider deleting this from these blank lines?